code/datums/mutations/actions.dm

```diff
@@ -190,3 +190,38 @@
 /obj/effect/proc_holder/spell/self/void/cast(mob/user = usr)
 	. = ..()
 	new /obj/effect/immortality_talisman/void(get_turf(user), user)
+
+/datum/mutation/human/self_amputation
+	name = "Autotomy"
+	desc = "Allows a creature to voluntary discard a random appendage."
+	quality = MINOR_NEGATIVE //upsides and downsides
```

Comment:
What exactly is the downside? There doesn't appear to be one unless you press the button.